### ISSUE TITLE
[bug]: avoid word break in the title of issue card

### ISSUE THREAD

**Issue by rlscode** [makeplane/plane#1216]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

When i put the the card view in Issues, the card break the word

### Steps to reproduce

1. Create new issue i suggest this text "Conectar el buscador con los filtros en la página de búsqueda"
2. Select the card view

![image](https://github.com/makeplane/plane/assets/25810274/e038979d-1661-4de0-ad53-83e37ec3d211)

### Browser

Google Chrome

### Version

Local

**Comment by rlscode** [comment]
This could be fixed with CSS

word-break: break-word;

**Comment by rhea0110** [comment]
Hey @rlscode, thank you for reporting this bug. We have already begun addressing it, and we plan to have it fixed and included in our upcoming release.

Thank you!

**Comment by aaryan610** [comment]
This issue will be fixed once PR #1339 is merged.

**Comment by aaryan610** [comment]
This issue has been fixed and is live on https://app.plane.so
